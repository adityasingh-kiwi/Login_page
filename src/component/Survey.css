.background-survey{
     height:800px;
     background-image: url("/home/kiwitech/Desktop/Project/login_page/src/image/Survey.jpg");
     background-size: cover;
     
     
    
}
.survey{
    display: flex;
    align-items: center;
    justify-content: center;
  
    color: darkorange;
    

    

}
.card-survey{
    position: relative;
    top: 250px;
    max-width: 700px;
    width: 450px;
    height: 100%;
    text-align: center;
    color: darkolivegreen;
    background-color: cadetblue;
}
 .is-info-is-medium{
    color: white;
    background-color: black;
}

label.radio{
    display: block;
    text-align: left;
    padding: 15px;
    margin-bottom: 5px;
    font-size: 20px;

}

label.radio+.radio{
    margin-left: 0;
}
label.radio input{
    margin-right: 10px;
}

.card-question{
    
    max-width: 700px;
    width: 450px;
    height: 100%;
    text-align: center;
    color: #333;
    background-color: cadetblue;
}
.survey-question{
    display: flex;
    align-items: center;
    justify-content: center;
  
    color: darkorange;
}
.background-start{
    height:800px;
    background-image: url("/home/kiwitech/Desktop/Project/login_page/src/image/Survey.jpg");
    background-size: cover;
}
