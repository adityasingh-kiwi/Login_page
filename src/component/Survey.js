import React,{useState,useEffect} from 'react';
import './Survey.css';
import RequestService from "./AddQuestionApi";




const Survey = ({onQuizStart}) => {


    



    return (
        <div className="background-survey">
        
        <div className="survey">
        <div className="card-survey">
            <div className="card-content">
                <div className="content">
                    <h1>Start the Survey</h1>
                    <p>Best of Luck</p>
                    <button className="button is-info-is-medium"onClick={()=>onQuizStart()}>Start</button>
                </div>
            </div>
           
        </div>
        </div>
        </div>
    )
}

export default Survey
