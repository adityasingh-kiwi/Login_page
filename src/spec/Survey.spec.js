import Survey from '../component/Survey'
import { mount, render, shallow, configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';


configure({ adapter: new Adapter() });

describe('Survey_page',()=>{
   it('Survey_button check',()=>{
        let wrapper= shallow(<Survey/>);
        expect(wrapper.exists('.content')).toEqual(true);
        expect(wrapper.length).toEqual(1)
        wrapper.find('.content').simulate('click');
   
   })
   it('Snapshot_Survey',()=>{
      let wrapper=shallow(<Survey/>)
      expect(wrapper).toMatchSnapshot();


    })

})