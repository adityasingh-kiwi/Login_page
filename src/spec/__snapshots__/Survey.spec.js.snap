// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Survey_page Snapshot_Survey 1`] = `
<div
  className="background-survey"
>
  <div
    className="survey"
  >
    <div
      className="card-survey"
    >
      <div
        className="card-content"
      >
        <div
          className="content"
        >
          <h1>
            Start the Survey
          </h1>
          <p>
            Best of Luck
          </p>
          <button
            className="button is-info-is-medium"
            onClick={[Function]}
          >
            Start
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
